<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayWindow</class>
 <widget class="QDialog" name="DisplayWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labDay">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>61</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>天</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>8</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/logo/res/PS.ico</pixmap>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdDay">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border:0px</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="intValue" stdset="0">
    <number>457</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMenu">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>更多…(Home)</string>
   </property>
   <property name="text">
    <string>…</string>
   </property>
   <property name="shortcut">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QLabel" name="labTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>341</width>
     <height>191</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>341</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>高考倒计时：</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>水滴时钟</string>
   </property>
  </widget>
  <action name="actExit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="actShutdown">
   <property name="text">
    <string>强制关机</string>
   </property>
   <property name="toolTip">
    <string>强制关机</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actRebbot">
   <property name="text">
    <string>强制重启</string>
   </property>
   <property name="toolTip">
    <string>强制重启</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actDisplay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>显示主界面</string>
   </property>
   <property name="toolTip">
    <string>显示主界面</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actNoForceReboot">
   <property name="text">
    <string>重启</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actNoForceShutdown">
   <property name="text">
    <string>关机</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actLogout">
   <property name="text">
    <string>注销</string>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
  </action>
  <action name="actForceLogout">
   <property name="text">
    <string>强制注销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
